integration-cli/docker_cli_run_test.go

```diff
@@ -3948,6 +3948,7 @@ func (s *DockerSuite) TestRunAttachFailedNoLeak(c *check.C) {
 	// TODO Windows Post TP5. Fix the error message string
 	c.Assert(strings.Contains(string(out), "port is already allocated") ||
 		strings.Contains(string(out), "were not connected because a duplicate name exists") ||
+		strings.Contains(string(out), "The specified port already exists") ||
```

Comment:
Tempted to say we should put comments for each error, documenting what version of Windows they match with (so that we can remove them once they're no longer needed) 😞